src/execution_plan/ops/op_sort.h

```diff
@@ -17,9 +17,9 @@ typedef struct {
 	heap_t *heap;               // Holds top n records.
 	Record *buffer;             // Holds all records.
 	uint limit;                 // Total number of records to produce, 0 no limit.
-	int direction;              // Ascending / desending.
+	int *directions;            // Ascending / desending.
```

Comment:
please re-write the comment, as `directions` is now the array of sorting directions